addons/website_sale/views/templates.xml

```diff
@@ -14,7 +14,7 @@
             <t t-set="show_cart" t-value="true"/>
             <li t-attf-class="#{_item_class} divider d-none"/> <!-- Make sure the cart and related menus are not folded (see autohideMenu) -->
             <li t-attf-class="o_wsale_my_cart #{not show_cart and 'd-none'} #{_item_class}">
-                <a href="/shop/cart" t-attf-class="#{_link_class}">
+                <a href="/shop/cart" t-attf-class="#{_link_class}" aria-label="Website sale cart">
```

Comment:
Aria label are for users no? "Website sale" is more technical, maybe `eCommerce cart`?